testing_and_setup/compass/ocean/global_ocean/EC60to30/init/config_E3SM_coupling_files.ini

```diff
@@ -14,9 +14,9 @@ prefix = EC
 description = MPAS Eddy Closure mesh for E3SM version ${e3sm_version} with
               enhanced resolution around the equator (${min_res} km) and poles
               (35 km), ${max_res}-km resolution at mid latitudes, and ${levels}
-              vertical levels
+              vertical levels.  Bathymetry is from GEBCO.
```

Comment:
@xylar I thought it would be good to add bathymetry to the metadata.  I don't think this is the greatest phrasing though.  I'd appreciate any suggestions.

Reply:
I totally agree.  Maybe add it in the other PR, too, if you haven't already.